pom.xml

```diff
@@ -327,19 +327,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-javadoc-plugin</artifactId>
-                <version>3.5.0</version>
-                <executions>
-                    <execution>
-                        <id>attach-javadocs</id>
-                        <goals>
-                            <goal>jar</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
```

Comment:
Please revert. JavaDoc is enforced for all modules. Otherwise, we cannot publish to mvn central :)

Reply:
Was probably accidentally committed, as it was a local change to circumvent a build problem.